utils/src/tree/GenericTreeItem.ts

```diff
@@ -28,3 +29,7 @@ export class GenericTreeItem extends AzExtTreeItem implements types.GenericTreeI
         return this._includeInTreeItemPicker;
     }
 }
+
+export function isGenericTreeItem(item: unknown): item is GenericTreeItem {
+    return (item as GenericTreeItem)._isGenericTreeItem;
```

Comment:
For maximum runtime safety?
```suggestion
    return typeof item === 'object' && (item as GenericTreeItem)._isGenericTreeItem;
```
(e.g. as written, if I passed in `undefined`, it would cause an error)